README.md

```diff
@@ -362,97 +372,193 @@ To get the first page of `ThreatDetections`, use the following query:
 ```console
 $ curl -H "Authorization: Bearer ${ACCESS_TOKEN}" \
        -H "Accept: application/json" \
-  https://api.cta.eu.amp.cisco.com/threat-detection/customer/{CUSTOMER_ID}/threat-detections
+  "https://api.cta.eu.amp.cisco.com/threat-detection/customer/${CUSTOMER_ID}/threat-detections"
 ```
 
-To get the first page of `Events`:
+To get the first page of security `Events`:
 
 ```console
 $ curl -H "Authorization: Bearer ${ACCESS_TOKEN}" \
        -H "Accept: application/json" \
-  https://api.cta.eu.amp.cisco.com/event-detection/customer/{CUSTOMER_ID}/events
+  "https://api.cta.eu.amp.cisco.com/event-detection/customer/${CUSTOMER_ID}/events"
 ```
 
-Each convicting `Event` can belong to one or more `ThreatDetections`. This can lead to processing a single convicting `Event` multiple times. However relation to multiple `ThreatDetections` is quite rare.
+### Modification Sequence Number
 
-Most convicting `Events` contain `securityAnnotation`, which represents key observations used for detecting threat or malicious behavior.
+Because there are potentially a lots of `ThreatDetections` and magnitude more `Events` we need to have mechanism
```

Comment:
These 2 paragraphs are not that great. Way too rambly for a documentation, many typos and bad grammar. I'll come back to these later with another proposal.